packages/asset-service/src/generateAssetData/generateAssetData.ts

```diff
@@ -17,8 +18,22 @@ import {
   getZapperTokens
 } from './ethTokens/extendErc20'
 
+const blacklistedAssets: Array<string> = blacklist
+
+const filterBlacklistedAssets = <T extends BaseAsset | TokenAsset>(assets: T[]) => {
```

Comment:
much much nicer, ty 🙏 as a bonus, you could add a unit test for this function if you like

Reply:
Ok, I will do it tomorrow, beginning to be late here in France :)

Reply:
I think this unit test doesn't belong to AssetService, I'll create a new file introducing unit tests to utils functions, right?